City-of-Helsinki/drupal-hdbt
# UHF-2879: Style 404, 403 and 401 pages, add illustrations for error a…

…nd login pages and maintenance page, add preprocess check if we are on error page and add class to body if so.

How to test:

1. Take any instance and set it up with latest dev branch.
2. `composer require drupal/hdbt:dev-UHF-2879_error_pages`
3. `make drush-cr`
4. `drush locale:check && drush locale:update`
5. Go to a url that doesn't exist. For example '/asfdgfkhlgutjfhgkj' is a good one. The 404 page should look nice now.
6. Log out and try to go to '/admin' for example. That 403 page should look nice as well. It should be 1:1 same as 401 page. If you know how to get into 401 page you can check it too :)
7. Test all error pages on all languages as well.

Attention! the login block on 403 & 401 page doesn't look similar to the layouts since it will be styled to the designed look on story https://helsinkisolutionoffice.atlassian.net/browse/UHF-3922
